core/controllers/base_test.py

```diff
@@ -258,6 +324,12 @@ def mock_get_current_time(unused_cls):
             self.assertFalse(base.CsrfTokenManager.is_csrf_token_valid(
                 'uid', token))
 
+    def test_redirect_oppia_test_server(self):
+        self.testapp.get(
```

Comment:
I think (a) this needs more explanation of what's going on since it's not clear to an observer, (b) we should be using self.get_json etc. rather than self.testapp.get().

Reply:
Done